### ISSUE TITLE
Failed to load the linter-gams package

### ISSUE THREAD

**Issue by Matzee** [chrispahm/linter-gams#29]
Hi chris, 

very nice work! Well done! :-) 

I updated the package and since then I unfortunately have an error raised from the vue framework. 

[Enter steps to reproduce:]

**Atom**: 1.37.0 x64
**Electron**: 2.0.18
**OS**: Ubuntu 18.04.2
**Thrown From**: [linter-gams](https://github.com/chrispahm/linter-gams) package 1.6.1

### Stack Trace

Failed to load the linter-gams package

```
At Cannot find module 'Vue/dist/vue.js'

Error: Cannot find module 'Vue/dist/vue.js'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/snap/atom/227/usr/share/atom/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:154765)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146072)
    at /packages/linter-gams/templates/main-component.js:3:21)
    at /packages/linter-gams/templates/main-component.js:387:3)
    at Module.get_Module._compile (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146072)
    at /packages/linter-gams/lib/gams-view.js:11:31)
    at /packages/linter-gams/lib/gams-view.js:5:35)
    at Module.get_Module._compile (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146072)
    at /packages/linter-gams/lib/main.js:4:23)
    at /packages/linter-gams/lib/main.js:208:2)
    at Module.get_Module._compile (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/snap/atom/227/usr/share/atom/resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
```

**Comment by chrispahm** [comment]
Hi @Matzee,

sorry for the inconvenience! 
I tested the update on Windows and macOS, but unfortunately I don't have access to a Linux machine. 
Could you do me a favour and manually check if the `vue.js` file exists in the `linter-gams/node_modules/Vue/dist` directory?
Best
Christoph

**Comment by chrispahm** [comment]
Thanks for the PR again! This should be fixed by https://github.com/chrispahm/linter-gams/pull/30, and the new patch released in v1.6.2!
